drivers/wifi/esp/esp_offload.c

```diff
@@ -237,8 +237,8 @@ static int _sock_send(struct esp_data *dev, struct esp_socket *sock)
 	k_sem_reset(&dev->sem_tx_ready);
 
 	ret = modem_cmd_send_nolock(&dev->mctx.iface, &dev->mctx.cmd_handler,
-			     NULL, 0, cmd_buf, &dev->sem_response,
-			     ESP_CMD_TIMEOUT);
+			     cmds, ARRAY_SIZE(cmds), cmd_buf,
```

Comment:
is this wrongly indented? (original lines look like it too actually). Or is it github code viewer? please check

Reply:
Yea they were wrongly indented :+1: Fixed.